gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -337,10 +337,10 @@ public class GoogleHadoopFileSystemConfiguration {
    * Minimum size in bytes of the HTTP Range header set in GCS request when opening new stream to
    * read an object.
    */
-  public static final HadoopConfigurationProperty<Integer> GCS_INPUT_STREAM_MIN_RANGE_REQUEST_SIZE =
+  public static final HadoopConfigurationProperty<Long> GCS_INPUT_STREAM_MIN_RANGE_REQUEST_SIZE =
       new HadoopConfigurationProperty<>(
           "fs.gs.inputstream.min.range.request.size",
-          GoogleCloudStorageReadOptions.DEFAULT_MIN_RANGE_REQUEST_SIZE);
+          (long) GoogleCloudStorageReadOptions.DEFAULT_MIN_RANGE_REQUEST_SIZE);
```

Comment:
Probably better change type of the `DEFAULT_MIN_RANGE_REQUEST_SIZE` class field to long too

Reply:
Yes - I tried that but some tests have hard dependency on this variable to be int. They are using this variable in creating ByteBuffer and that takes int as capacity.

Reply:
We could use `toIntExact` similar to https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadChannel.java#L808

+1 to updating the default to long instead

Reply:
Fixed it. Thanks!